virt/virt-4-11-release-notes.adoc

```diff
@@ -131,15 +128,15 @@ Removed features are not supported in the current release.
 +
 To preserve and support your existing nmstate configuration, install the xref:../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc#k8s-nmstate-about-the-k8s-nmstate-operator[Kubernetes NMState Operator] before updating to {VirtProductName} {VirtVersion}. You can install it from the *OperatorHub* in the {product-title} web console, or by using the OpenShift CLI (`oc`).
 
-* As of version {VirtVersion}, the `node-maintenance-operator` (NMO) is no longer shipped with {VirtProductName}. It is now available to deploy as a standalone Operator from the *OperatorHub* in the {product-title} web console.
+* As of version {VirtVersion}, the Node Maintenance Operator (NMO) is no longer shipped with {VirtProductName}. You can xref:../nodes/nodes/eco-node-maintenance-operator.adoc#node-maintenance-operator[install the NMO] from the *OperatorHub* in the {product-title} web console, or by using the OpenShift CLI (`oc`).
```

Comment:
Do you think this RN might persist in future RNs? If so, I think `{VirtVersion}` should be changed to a hard-coded "4.11". If not, I am not sure that the phrase `As of version {VirtVersion}` is necessary.

Reply:
This RN will not persist in future RNs. I will remove `As of version {VirtVersion}`.